settings.gradle

```diff
@@ -25,7 +25,7 @@ if (file('local.properties').canRead()) {
     localProperties.load(file('local.properties').newDataInputStream())
     log('Loaded local.properties')
 } else {
-    log('Missing local.properties; see https://github.com/mozilla-mobile/focus-android/blob/master/README.md#local-properties-helpers for instructions.')
+    log('Missing local.properties; see https://github.com/mozilla-mobile/focus-android/blob/master/README.md#local.properties helpers for instructions.')
```

Comment:
**issue (blocking):** I don't think autolinking of the URL will work because there's a space in the anchor between "properties" and "helpers"

**suggestion:** The fenix link is https://github.com/mozilla-mobile/fenix#localproperties-helpers – should we have a similar ending here?